PCA_Comp_Analysis_Dredging.R

```diff
@@ -322,67 +442,41 @@ X_nonas<-X[ , colSums(is.na(X)) == 0]
 ncol=dim(X_nonas)[2]
 Xmatrix<-as.matrix(X_nonas[,2:ncol])
 
-
-######################################################################
-#run PCA 
-######################################################################
-#run PCA on the rainfall data and retain the eigenvectors of the first 10 PCs
+###############################################################
+# (2.3) Run PCA on rainfall data (predictands) 
+###############################################################
+#run PCA on the rainfall data and retain the 
+# eigenvectors of the first 10 PCs
 
 pr<-prcomp(Xmatrix,center=T,scale=T)
 
-
 # eigenvectors of the PCA (weight factors for the stations) 
 eigenvectors<-pr$rotation
 
 notScaled=eigenvectors[,1:10]

```

Comment:
we only use 4 of these 10 selected eigenvectors - should we change this to just 4? I think we had used 10 in making the north rule box plots just to verify that 4 was sufficient

Reply:
In the script _nonScaled_ is not used down below in the code. _pcacomp_ can be 1,2,3, or 4 in our case. However, in latest changes I made _pcacomp_ a user variable, and introduced _pca_max=4_ as the maximum number of modes since we have only 4 models defined for the dredge-function. I updated that part where we define variable _res_ for the dredge-function.